Gemfile

```diff
@@ -169,7 +169,7 @@ group :test, :development do
   gem 'rspec-rails', '~> 3.4.0'
   gem 'pry', '~> 0.10.0'
   gem 'pry-debugger', '~> 0.2.0', :platforms => :ruby_19
-    gem 'public_suffix', '< 1.5.0'
+    gem 'public_suffix', '~> 1.4.0', '< 1.5.0'
```

Comment:
nope, bundler we rely on is too old so doesn't support the latest ruby platforms. sigh.